feature/gnoi/system/tests/kill_a_daemon_test/README.md

```diff
@@ -0,0 +1,32 @@
+# gNOI-3.6: Kill a Daemon Test
```

Comment:
To differentiate from gNOI-3.5 which also kills a daemon and uses healthz:

```suggestion
# gNOI-3.6: Configuration daemon kill process  and healthz
```

Reply:
Incorporated in [copying_debug_files_test](https://github.com/openconfig/featureprofiles/pull/2271) instead.